drivers/pinctrl/pinctrl_ti_k3.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Enphase Energy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_k3_pinctrl
```

Comment:
It looks like this macro isn't used here

Reply:
Fixed